apps/storybook/src/Tabs.stories.tsx

```diff
@@ -11,12 +11,12 @@ import {
   HorizontalTabsProps,
   VerticalTabsProps,
   Tab,
+  TabsProps,
 } from '@itwin/itwinui-react';
-import { Tabs, TabsProps } from '@itwin/itwinui-react/esm/core/Tabs/Tabs';
 
 export default {
   title: 'Core/Tabs',
-  component: Tabs,
+  component: HorizontalTabs,
```

Comment:
Just curious, do we keep this `Tabs` since this story shows both Horizontal and Vertical tabs? Also to confirm, what does this `component` prop actually do?